assets/js/blocks/mini-cart/mini-cart-contents/inner-blocks/mini-cart-title-items-counter-block/block.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "woocommerce/mini-cart-title-items-counter-block",
+	"version": "1.0.0",
+	"title": "Mini Cart Title Items Counter",
+	"description": "Block that displays the items counter part of the Mini Cart Title block.",
+	"category": "woocommerce",
+	"supports": {
+		"align": false,
+		"html": false,
+		"multiple": false,
+		"reusable": false,
+		"inserter": true,
```

Comment:
We will need to add `"lock": false`, otherwise I can unlock it and remove it. Also, we should probably set `"inserter": false` too.

![imatge](https://user-images.githubusercontent.com/3616980/231416858-47ad449f-9c02-47a0-a5f2-7e42f51e580f.png)

Reply:
Done!